### PR TITLE
Add HTTP output

### PR DESCRIPTION
**Pull request by H4ckd4ddy**
Hi

First, thanks for your great tool.
It could be useful to add basic HTTP output

Best regards

### CODE REVIEW

File: settings.json.sample [hunk 1]
```diff
@@ -130,6 +130,15 @@
       "webhook_url": "",
       "rule_list": ["custom_keywords"]
     },
+    "http_output": {
+      "enabled": false,
+      "module": "outputs.http_output",
+      "classname": "HttpOutput",
+      "endpoint_url": "",
+      "http_auth": false,
```
Comment (Plazmaz) [inline settings.json.sample]: I think a more elegant solution would be setting this up to be a headers dictionary, where you could set stuff like basic auth if you wanted to, but could also use this output for other outputs (such as a splunk HEC or a webhook).

File: pastehunter/outputs/http_output.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        if self.endpoint_url == '':
+            logging.error("HTTP endpoint not configured")
+            self.valid = False
+
+    def store_paste(self, paste_data):
+        if self.valid:
+            
+            json_data = paste_data
+            del json_data['raw_paste']
```
Comment (Plazmaz) [inline pastehunter/outputs/http_output.py]: This seems like something that should be configurable and not enabled by default (maybe `ignore_fields` in the http config)?

File: pastehunter/outputs/http_output.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            json_data['@timestamp'] += 'Z'
```
Comment (Plazmaz) [inline pastehunter/outputs/http_output.py]: I am guessing this is for a specific use case? I don't think adding the Z is generally applicable for this.

File: pastehunter/outputs/http_output.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            if self.http_password:
+                #req = requests.post(self.endpoint_url, headers=headers, data=json_data, auth=(self.http_user, self.http_password))
```
Comment (Plazmaz) [inline pastehunter/outputs/http_output.py]: Please remove this commented code.

File: pastehunter/outputs/http_output.py (hunk 2 above)
Comment (H4ckd4ddy) [reply pastehunter/outputs/http_output.py]: your right, i just added it

File: pastehunter/outputs/http_output.py (hunk 3 above)
Comment (H4ckd4ddy) [reply pastehunter/outputs/http_output.py]: It is to respect the format of ISO 8601, I had set GMT by default, but I have just added a setting to define the timezone

File: pastehunter/outputs/http_output.py (hunk 4 above)
Comment (H4ckd4ddy) [reply pastehunter/outputs/http_output.py]: Sorry, I committed a little quickly, I don't often make a pull request :)

File: settings.json.sample (hunk 1 above)
Comment (H4ckd4ddy) [reply settings.json.sample]: I added an array of headers in the settings, but I think it is useful to leave the username / password options for auth basics, which avoids preparing the header

**Comment by H4ckd4ddy** [comment]
Thanks for your responses, I made some modifications

**Pull request merged**